fedimg/services/ec2/ec2imgpublisher.py

```diff
@@ -152,20 +150,7 @@ def get_volume_type_from_image(self, image):
         return blk_mapping[0]['ebs']['volume_type']
 
     def get_virt_type_from_image(self, image):
-        if isinstance(image, str):
-            image_id = image
-            image = self._connect().get_image(image_id)
-
-        blk_mapping = image.extra['block_device_mapping']
-        if not blk_mapping:
-            blk_mapping = self._retry_till_blk_mapping_is_available(image)
- 
-        device_name = blk_mapping[0]['ebs']['volume_type']
-
-        if device_name.endswith('sda1'):
-            return 'hvm'
-        else:
-            return 'paravirtual'
+        return "hvm"
```

Comment:
maybe use single quotes here rather than double quotes, since everywhere else seems to use single quotes.